lib/model/CTokenListCategory.cc

```diff
@@ -178,6 +178,10 @@ bool CTokenListCategory::acceptRestoreTraverser(core::CStateRestoreTraverser& tr
         }
     } while (traverser.next());
 
+    // Ensure that m_CommonUniqueTokenIds is sorted in ascending order.
+    std::sort(m_CommonUniqueTokenIds.begin(), m_CommonUniqueTokenIds.end(),
+              [](const auto& a, const auto& b) { return a.first < b.first; });
```

Comment:
Currently this is the same but presumably if `CTokenIdLess` changed this would need to too:
```suggestion
             CTokenIdLess{});
```